modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/select_account_users.jsp

```diff
@@ -30,8 +30,24 @@ if (selectAccountUsersManagementToolbarDisplayContext.isSingleSelect()) {
 String eventName = ParamUtil.getString(request, "eventName", liferayPortletResponse.getNamespace() + "assignAccountUsers");
 %>
 
-<clay:management-toolbar-v2
-	displayContext="<%= selectAccountUsersManagementToolbarDisplayContext %>"
+<liferay-portlet:renderURL portletName="<%= AccountPortletKeys.ACCOUNT_USERS_ADMIN %>" var="addAccountEntryUserURL">
+	<portlet:param name="mvcRenderCommandName" value="/account_admin/add_account_user" />
+	<portlet:param name="redirect" value='<%= ParamUtil.getString(request, "redirect") %>' />
+	<portlet:param name="backURL" value='<%= ParamUtil.getString(request, "redirect") %>' />
+	<portlet:param name="accountEntryId" value='<%= ParamUtil.getString(request, "accountEntryId") %>' />
+</liferay-portlet:renderURL>
+
+<clay:management-toolbar
+	additionalProps='<%=
+		HashMapBuilder.<String, Object>put(
+			"addAccountEntryUserURL", addAccountEntryUserURL.toString()
+		).put(
+			"openModalOnRedirect", selectAccountUsersManagementToolbarDisplayContext.isOpenModalOnRedirect()
+		).build()
+	%>'
+	managementToolbarDisplayContext="<%= selectAccountUsersManagementToolbarDisplayContext %>"
+	propsTransformer="account_entries_admin/js/SelectAccountUsersManagementToolbarPropsTransformer"
+	showCreationMenu="<%= selectAccountUsersManagementToolbarDisplayContext.isShowCreateButton() %>"
```

Comment:
I guess this should be inside the displayContent, there is a `isShowCreationMenu` method...

Reply:
@ambrinchaudhary  It may be that the product team forgot to move this to `DisplayContext`, but this also may be intentional. For example, `selectAccountUsersManagementToolbarDisplayContext` may be used by multiple management toolbars. If that is the case, this would be a way of overriding the default.

Reply:
I only added that there because in the file previously, the toolbar handler was wrapped in an `if` check on whether to be included or not.  See https://github.com/liferay-frontend/liferay-portal/pull/807/files/dd79ff48f9feddfb1704a083527b10adf8352bab..2dfcfea9d82435381e6ab7d871e5123eac8e62e8#diff-eaf417c5b06d208194a5c763e75fa438627504d8a480a75851ec63dd2bc61506L135.

I figured since we are migrating everything and the toolbar always needs to show, I would try to migrate their existing logic over, but if you feel it is better to have this in the `DisplayContext` and utilize the `isShowCreationMenu` method I can move it there.

Reply:
If this was like this before, I'm ok, sorry for the confusion, I didn't see how was it previously

Reply:
No problem.  Just wanted to make sure you understood the reasoning behind the change.  I am open to your suggestions on how to make things better, so these conversations are very good to me to think twice about everything we are doing and find ways to improve.